### PR TITLE
Celebrations 300g added

### LINKED ISSUE #60: Add CELEBRATIONS (300g)
We need somebody to add the product **CELEBRATIONS (300g)** (this is the product name) to the project.

This product can be categorized into the **Product Type** of _Food, Drinks & Beverages._
Here is a helpful link:
http://www.marschocolate.com.au/products/celebrations/celebrations-gift-box-300g/

Read the README.md and CONTRIBUTING.md before making a PR and contributing.

Thanks for taking the time to contribute to our project, we appreciate your help! 🌟

### PR DESCRIPTION
**Pull request by PallaviDas07**
Fixes #60 
Celebrations 300g(U.K., Egypt, Saudi Arabia, U.A.E, 06.10.17) Added

### CODE REVIEW

File: Celebrations 300g (U.K., Egypt, Saudi Arabia, U.A.E., 06.10.2017)/celebrations.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Product_Type; Food, Drinks & Beverages
+
+Specific_Product_Type; Milk Chocolate
+
+Ingredients; Sugar, Milk Solids, Glucose Syrup (Sources Include Wheat), Cocoa Butter, Cocoa Mass, Peanuts, Desiccated Coconut, Vegetable Fat, Emulsifiers (Soy Lecithin, 471), Barley Malt Extract, Salt, Cocoa Powder, Humectant (Glycerol), Egg White, Flour (Sources include Wheat), Flavours, Wheat Gluten, Raising Agents (500, 501).
+
+Nutritional_Value;Protein(5.6 g), Carbohydrate(62 g), Fat(24.2 g), Fibre(1.2 g)/ 100 g
+
+Energy; 502kcal (2100kJ) / 100 g
```
Comment (robertoduessmann) [inline Celebrations 300g (U.K., Egypt, Saudi Arabia, U.A.E., 06.10.2017)/celebrations.txt]: Here wouldn't be 502**cal**?

Comment (KAUTH) [reply Celebrations 300g (U.K., Egypt, Saudi Arabia, U.A.E., 06.10.2017)/celebrations.txt]: @PallaviDas07 yes it would be 502cal

Comment (KAUTH) [reply Celebrations 300g (U.K., Egypt, Saudi Arabia, U.A.E., 06.10.2017)/celebrations.txt]: @PallaviDas07 also keep in mind that we put a picture of the product's label and not the packaging, thanks.

**Comment by KAUTH** [comment]
@PallaviDas07 there were a few corrections, check them out. Thanks for contributing!!

**Comment by PallaviDas07** [comment]
I'll make the necessary changes. Thank you.

**Comment by KAUTH** [comment]
@PallaviDas07 I corrected them, just a reminder for next time. Thanks!

**Comment by PallaviDas07** [comment]
Thank you very much!

**Pull request merged**